### PR TITLE
HW/memeoryGame

### PR DESCRIPTION
**Pull request by jbuican14**
1. index, game_work.js and query file
2. Game has reset button
3. Need other functionalities like ‘score’, configurable board size

### CODE REVIEW

File: game_work.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                test2.className = '';
+              }
+            }
+          } else {
+            prevSelectedValue = imgValue;
+            evt.currentTarget.parentNode.className = 'on';
+          }
+        });
+      }
+    },
```
Comment (afeld) [inline game_work.js:88]: How could you split up this method?

File: index.html [hunk 2]
```diff
@@ -1,8 +1,44 @@
...
+
+      <li data = "2"><a href="#"><img class="hid8" src="assets/imac.png" alt="oypad"><img src="assets/open-box.png" alt="wine"></a></li>
+      <li data = "4"><a href="#"><img class="hid9" src="assets/keyboards.png" alt="target"><img src="assets/open-box.png" alt="wine"></a></li> 
+      <li data = "3"><a href="#"><img class="hid10" src="assets/diamond.png" alt="man"><img src="assets/open-box.png" alt="wine"></a></li>
+      <li data = "1"><a href="#"><img class="hid11" src="assets/speakers.png" alt="wine"><img src="assets/open-box.png" alt="wine"></a></li>
+
+      <li data = "2"><a href="#"><img class="hid12" src="assets/diamond.png" alt="oypad"><img src="assets/open-box.png" alt="wine"></a></li>
+      <li data = "4"><a href="#"><img class="hid13" src="assets/speakers.png" alt="target"><img src="assets/open-box.png" alt="wine"></a></li> 
+      <li data = "3"><a href="#"><img class="hid14" src="assets/keyboards.png" alt="man"><img src="assets/open-box.png" alt="wine"></a></li>
+      <li data = "1"><a href="#"><img class="hid15" src="assets/imac.png" alt="wine"><img src="assets/open-box.png" alt="wine"></a></li>
```
Comment (afeld) [inline index.html:40]: How could you avoid needing to hard-code all of these?

**Comment by afeld** [comment]
Hmm, looking at http://jbuican14.github.io/memory/, it seems that I can't actually click any tiles to match them. Maybe I'm missing something?

The code looks good, though I would like to see you make a pass at refactoring to make it easier to follow. Thanks!

File: index.html (hunk 2 above)
Comment (jbuican14) [inline index.html:40]: I actually have an idea of using js generating the DOM. I was  able to do so in the homework for ajax at www.jb012.com/mashup  that i am going to submit it very very soon.

**Comment by jbuican14** [comment]
Hi @afeld,

I was surprised to see that the code did not let the game begin. here is also where I put my game:
http://www.jb012.com/memory

**Comment by afeld** [comment]
That works, thanks!

**Pull request closed without merging**